src/_common/video/player/player.ts

```diff
@@ -50,24 +60,56 @@ function createReadableTimestamp(time: number) {
 		AppPlayerScrubber,
 		AppPlayerFullscreen,
 		AppShortkey,
+		AppResponsiveDimensions,
+		AppImgResponsive,
+		AppMediaItemBackdrop,
+	},
+	directives: {
+		AppTooltip,
 	},
 })
 export default class AppVideoPlayer extends Vue {
-	@Prop(propRequired(String)) poster!: string;
+	@Prop(propRequired(MediaItem)) mediaItem!: MediaItem;
 	@Prop(propRequired(Array)) manifests!: string[];
 	@Prop(propOptional(Boolean, false)) autoplay!: boolean;
 	@Prop(propOptional(Number, 0)) startTime!: number;
 	@Prop(propOptional(String, null)) context!: VideoPlayerControllerContext;
+	@Prop(propOptional(Number, 0)) viewCount!: number;
+	@Prop(propOptional(Boolean, false)) showVideoStats!: boolean;
 
-	player = new VideoPlayerController(this.poster, this.manifests, this.context);
-	isHovered = false;
+	player = new VideoPlayerController(undefined, this.manifests, this.context);
```

Comment:
remove the poster field since I think we always manually put it behind the video now